src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        Random random = new Random();
```

Comment:
It shouldn't be a local variable, cause each time you call getRandomColor() method, you will create new Random object, it's redundant

Reply:
Yeah, make sense. Dummy mistake, changed